install.sh

```diff
@@ -0,0 +1,7 @@
+#/bin/bash
+
+git clone https://github.com/PurpLabs/PogBot.git
+cd PogBot
+pip install -r requirements.txt
+cd src
+python3 main.py
```

Comment:
we shouldn’t run the file yet, because the user hasn’t set their environment variables yet (maybe include a message about that?)